medschool/Cargo.toml

```
@@ -26,6 +26,8 @@ clap = { version = "4", features = ["derive"] }
 tracing.workspace = true
 tracing-subscriber.workspace = true
 regex.workspace = true
+jsonschema = "0.18.0"
+serde_json = "1.0.122"
```

```suggestion
jsonschema = "0.18"
serde_json = "1.0"
```

We usually just use major.min everywhere else right?